gtests/net/mptcp/add_addr/add_addr_server.pkt

```diff
@@ -17,9 +17,8 @@
 +0     accept(3, ..., ...) = 4
 
 // ADD_ADDR is sent directly (>= 5.12), not after first data (< 5.12)
-+0       >   .  1:1(0)        ack 1              <add_address addr[saddr] hmac=auto, dss dack4=1           ssn=1 dll=0 nocs>
-// TODO: support injecting ADD_ADDR echo
-//+0     <   .  1:1(0)        ack 1     win 257  <add_address addr[saddr],           dss dack8=1           ssn=1 dll=0 nocs>
++0       >   .  1:1(0)        ack 1              <add_address address_id=1 addr[ep=inet_addr("198.51.100.2")] hmac=auto, dss dack4=1 ssn=1 dll=0 nocs>
++0       <   .  1:1(0)        ack 1     win 257  <add_address address_id=1 addr[ep=inet_addr("198.51.100.2")] addr_echo,      dss dack8=1 ssn=1 dll=0 nocs>
```

Comment:
I don't know if I can change your PR because you have maybe already modified it to be on top of PR #44 and #45 but if you don't mind, may you keep the alignment of the dss options here above please? :)

Reply:
whoops, totally forgot about it. Sure, will fix it

Reply:
done